### ISSUE TITLE
admin.conf client certificates expired in sandbox and staging

### ISSUE THREAD

**Issue by nkinkade** [m-lab/k8s-support#446]
Today we discovered that the certificate in admin.conf had expired over the weekend. The admin.conf kubeconfig is used by operators, the root user on API server nodes, and also Cloud Build deployments.  This highlighted the fact that we need to make sure that certificates in the cluster do no expire, or that we are at least warned when it is about to happen.

[Apparently kubeadm has a flag, `--certificate-renewal`](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/#automatic-certificate-renewal), that will cause it to automatically renew all certificates whenever the node/cluster is upgraded. This didn't save us here because:

```
**Warning**: Prior to kubeadm version 1.17 there is a bug where the default value for --certificate-renewal is false for the kubeadm upgrade node command. In that case, you should explicitly set --certificate-renewal=true.
```
However, even if it is supposed to be the default, we should be explicitly setting this flag for good measure (as the bug proves). 

What still needs to be proven is that `--certificate-renewal=true` will also upgrade the admin.conf file, and even if it does we need to be sure that the updated certificate makes it into GCS.

**Comment by nkinkade** [comment]
Next time we upgrade k8s in our clusters we need to verify that certificate renewal is working as expected, including making sure that admin.conf get updated, and that admin.conf gets uploaded to GCS as part of the upgrade process because we have numerous scripts that expect to find a valid admin.conf there.

**Comment by nkinkade** [comment]
I have now verified that admin.conf gets upgraded along with every other certificate and key during the API cluster upgrade process. I will submit a PR which, among other things, will upload the renewed admin.conf credentials file to GCS.

**Comment by nkinkade** [comment]
This issue should be resolved by #463, which copies the updated admin.conf file to GCS after it is renewed.
